variables.tf

```diff
@@ -197,8 +197,16 @@ variable "reloader_custom_values" {
 variable "reloader_image" {
   type        = string
   description = "The reloader image in the format of `[registry-url]/[namespace]/[image]`."
+  default     = "stakater/reloader"
```

Comment:
This is very confusing. In main branch the value of `reloader_image` is "ghcr.io/stakater/reloader", which is the format that is in the variable description. But now you have added a second input called `reloader_repository` which now has the default value of "ghcr.io/stakater/reloader".

So to me that looks like you are trying to pul the image like this:
`ghcr.io/stakater/reloader/stakater/reloader`

When really it should be:
`ghcr.io/stakater/reloader`

But it seems that is what they are expecting in their helm chart :( -> https://github.com/stakater/Reloader/blob/0027bd73fd436785554360930027327eb75add90/deployments/kubernetes/chart/reloader/values.yaml#L18-L19

I wonder if we can handle this any better in our inputs? The reloader and ESO inputs are now not consistent.